EssentialsXMPP/src/com/earth2me/essentials/xmpp/Commandxmpp.java

```diff
@@ -25,7 +25,9 @@ protected void run(final Server server, final CommandSource sender, final String
         }
 
         final String message = getFinalArg(args, 1);
-        final String senderName = sender.isPlayer() ? ess.getUser(sender.getPlayer()).getDisplayName() : Console.NAME;
+        final String senderName = sender.isPlayer()
+                ? ess.getUser(sender.getPlayer()).getDisplayName()
+                : Console.DISPLAY_NAME;
```

Comment:
please keep this to one line

Reply:
I don't see any line lengths/wrapping guidelines in #3701, so I used common sense here -- wrapped everything which was above 120 symbols. A successful build indicates that no checkstyle rule was violated. I don't mind applying required length if there's any project-scope preference, not a personal opinion.
@md678685, any thoughts on this?

Reply:
Pretty sure the Essentials code doesn't split lines unless creating a builder class, I could be wrong however.

Reply:
Yeah just as a convention, I don't think Essentials normally wraps lines. Just because no checkstyle rule was violated (it was also added extremely recently) doesn't mean we can't still discuss it. 😄 Maybe let MD comment on this since he's managing those rules but I think one line is more correct here as far as I know.

Reply:
@mdcfe bump?

Reply:
One line is preferable where possible - many of these line breaks feel arbitrary.